### PR TITLE
doc: Update known issues for 1.5.1

### PR DESCRIPTION
**Pull request by umapraseeda**
Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -752,7 +753,7 @@ Closing sockets
 Multiprotocol Service Layer (MPSL)
 ==================================
 
-.. rst-class:: v1-5-0 v1-4-2 v1-4-1
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1
 
 DRGN-15223: `CONFIG_SYSTEM_CLOCK_NO_WAIT` is not supported for nRF5340
```
Comment (rugeGerritsen) [inline doc/nrf/known_issues.rst:758]: @hurricaneFromMoscow , DRGN-15223 is fixed for NCS 1.5.1, right?

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -816,7 +817,7 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
-.. rst-class:: v1-5-0
+.. rst-class:: v1-5-1 v1-5-0
```
Comment (rugeGerritsen) [inline doc/nrf/known_issues.rst:820]: DRGN-15382 is fixed in NCS 1.5.1. Remove "(once available)" from the workaround

**Review by rugeGerritsen — CHANGES_REQUESTED** [review]
Reviewed the section "SoftDevice Controller"

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst:835]: Updated

**Review by nordic-hani — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -355,19 +363,40 @@ Subsystems
 Bluetooth LE
 ============
 
-.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```
Comment (joerchan) [inline doc/nrf/known_issues.rst:366]: Fixed in 1.5.1

File: doc/nrf/known_issues.rst [hunk 4]
```diff
@@ -355,19 +363,40 @@ Subsystems
...
+
+DRGN-15435: GATT notifications and Writes Without Response might be sent out of order
+  GATT notifications and Writes Without Response might be sent out of order when not using a complete callback.
+
+  **Workaround:** Always set a callback for notifications and Writes Without Response.
+
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15448: Incomplete bond overwrite during pairing procedure when peer is not using the IRK stored in the bond
```
Comment (joerchan) [inline doc/nrf/known_issues.rst:375]: Fixed in 1.5.1

**Review by jhn-nordic — APPROVED** [review]

**Review by osaether — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (kromancer) [reply doc/nrf/known_issues.rst:794]: Yes, the fix is present on the v1.5-branch

**Review by rugeGerritsen — APPROVED** [review]
Reviewed the sections "SoftDevice Controller" and "MPSL"

File: doc/nrf/known_issues.rst (hunk 1 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst]: Updated

File: doc/nrf/known_issues.rst (hunk 3 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst]: Updated

File: doc/nrf/known_issues.rst (hunk 4 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst:375]: Updated

**Review by grochu — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
CHIP addition looks OK.

**Review by tomchy — APPROVED** [review]

**Comment by umapraseeda** [comment]
@frkv @carlescufi @LuDuda @hakonfam

**Comment by joerchan** [comment]
Why is it still in draft?

**Review by kromancer — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Review by LuDuda — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 5]
```diff
@@ -277,45 +285,110 @@ KRKNWK-6408: ``diag`` command not supported
...
+                            zb_buf_set_status(bufid, -1);
+                            status = -1;
+                     }
+             }
+             break;
+     }
+
+.. rst-class:: v1-5-1
+
+KRKNWK-9461 / KRKNWK-9596 : Multiprotocol samples crashes with some smartphones
```
Comment (MarekPorwisz) [inline doc/nrf/known_issues.rst:345]: This is also applicable for Thread multiprotocol samples.

**Review by joerchan — APPROVED** [review]

**Review by czeslawmakarski — APPROVED** [review]

File: doc/nrf/known_issues.rst (hunk 5 above)
Comment (umapraseeda) [reply doc/nrf/known_issues.rst]: Added the issue in the Thread section also.

**Review by MarekPorwisz — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Review by hakonfam — APPROVED** [review]

**Review by jtguggedal — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Pull request merged**